src/css/application.css

```diff
@@ -292,6 +295,57 @@ body {
   flex: 1 0 0;
 }
 
+/** @define sub-bars */
+
+.sub-bar {
+  box-shadow: var(--box-shadow);
+  font-size: var(--font-size-menu);
```

Comment:
I think we need a new name for this font size variable!

Reply:
Perhaps. It's becoming a general goto "base" font-size.